src/main/java/frc/robot/subsystems/Climber.java

```diff
@@ -22,6 +22,13 @@ public class Climber extends SubsystemBase {
   private final SparkPIDController m_leftPIDController;
   private final SparkPIDController m_rightPIDController;
 
+  private enum State {
+    PRE_FIRST_USE,
+    AFTER_FIRST_USE
+  }
+
+  private final State m_state;
```

Comment:
This can't be `final`. Assuming the code compiles, this alone tells us there's a bug in the state transition logic.